sites/public/src/lib/helpers.tsx

```diff
@@ -220,6 +214,17 @@ export const getImageCardTag = (listing: Listing): ImageTag[] => {
     : null
 }
 
+export const getApplicationSeason = (listing: Listing) => {
+  let label = t("listings.apply.applicationSeason")
+  if (listing?.marketingSeason) {
+    label = label.concat(` ${t(`seasons.${listing.marketingSeason}`)}`)
+  }
+  if (listing?.marketingDate) {
+    label = label.concat(` ${dayjs(listing.marketingDate).year()}`)
+  }
+  return label
+}
```

Comment:
I think it works for this case in the languages we support, but there is a chance that a language wouldn't have the same order in the sentence.
For example we might say "Residents should apply in Spring 2023" but another language it could be more appropriate to say "In Spring 2023 residents should apply". 
I don't think we have to do it for this one, but usually with these translations you should pass the season and year as parameters to the translation string.

Reply:
@ColinBuyck my assumption would also be to include the message even when no season is listed although @sarahlazarich might have a different recommendation.

Font weight discrepancy seems to be between the figma component and what you have in the code, but lets keep it 500 for consistency with the rest of the application. I added the free font awesome version to the mockup (solid) for access, but if you are using the hollow one consistently, then go ahead and use it here as well. Thanks for double checking!